### PR TITLE
Avoid to have focused tests in using eslint-plugin-jasmine

### PR DESCRIPTION
**Pull request by calixteman**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "eslint-plugin-fetch-options": "^0.0.5",
     "eslint-plugin-html": "^7.1.0",
     "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-jasmine": "^4.1.3",
```
Comment (Snuffleupagus) [inline package.json:24]: Looking at https://github.com/tlvince/eslint-plugin-jasmine it hasn't been updated for over two years, but I'm assuming that there's no more "maintained" alternative available?

File: test/.eslintrc [hunk 2]
```diff
@@ -1,10 +1,19 @@
...
+  "plugins": ["jasmine"],
+  "rules": {
+    "jasmine/no-focused-tests": "error",
+    "jasmine/new-line-before-expect": "off",
+    "jasmine/no-pending-tests": "off",
+    "jasmine/new-line-between-declarations": "off",
+    "jasmine/no-spec-dupes": "off",
+    "jasmine/prefer-jasmine-matcher": "off",
+    "jasmine/no-suite-dupes": "off",
+    "jasmine/prefer-toHaveBeenCalledWith": "off"
```
Comment (Snuffleupagus) [inline test/.eslintrc:17]: Can we please sort the rules alphabetically instead?

File: test/.eslintrc
Comment (Snuffleupagus) [inline test/.eslintrc:1]: For overall consistency, can we please sort the contents of this file in the same order as in https://github.com/mozilla/pdf.js/blob/master/.eslintrc?

So, basically something like:
```json
{
  "plugins": [
    ...
  ],

  "extends": [
    ...
  ],

  "env": {
    ...
  },

  "rules": {
    ...
  },
}
```

File: package.json (hunk 1 above)
Comment (calixteman) [reply package.json:24]: Yes I noticed that too but I can imagine that it's stable enough to not require any updates.
I didn't find any alternatives.

**Review by timvandermeij — APPROVED** [review]
Looks good me me; thank you for doing this! In this particular case I'm not too worried about the period without updates for the plugin because it doesn't seem to pull in any (pinned) dependencies itself which is usually what is causing problems.

**Review by Snuffleupagus — APPROVED** [review]

**Pull request merged**